flytekit/interaction/click_types.py

```diff
@@ -305,10 +305,12 @@ def __init__(self, python_type: typing.Type):
         self._python_type = python_type
 
     def _parse(self, value: typing.Any, param: typing.Optional[click.Parameter]):
-        if type(value) == dict or type(value) == list:
+        if isinstance(value, dict) or isinstance(value, list):
             return value
```

Comment:
why change `type` to `isinstance`?

Reply:
because I got a hint from the editor
<img width="666" alt="image" src="https://github.com/user-attachments/assets/686da1f7-31c8-4726-add3-b58f0f8a807a" />